ci-operator/step-registry/telcov10n/system/telco-kpis/run-cyclictest-test/telcov10n-system-telco-kpis-run-cyclictest-test-commands.sh

```
@@ -0,0 +1,137 @@
+    if [[ -d "${MOUNTED_GROUP_INVENTORY}/${SPOKE_CLUSTER}" ]]; then
+        find "${MOUNTED_GROUP_INVENTORY}/${SPOKE_CLUSTER}/" -mindepth 1 -type d | while read -r dir; do
+            echo "Process spoke group inventory file: ${dir}"
+            process_inventory "$dir" /eco-ci-cd/inventories/ocp-deployment/group_vars/"$(basename "${dir}")"
+        done
+    fi
+
+    echo "Create host_vars directory"
+    mkdir -p /eco-ci-cd/inventories/ocp-deployment/host_vars
+
+    # Copy spoke credentials to temporary location (hypervisor may be shared across spokes)
+    mkdir -m 700 -p /tmp/"${SPOKE_CLUSTER}"
```

Shellcheck (0.11.0)

[warning] When used with -p, -m only applies to the deepest directory.

(SC2174)